framework/interface/templates/target.html

```diff
@@ -130,252 +25,62 @@ <h3 id="rankingTargetInfo"></h3>
-                <label for="name">{{ filter_type.replace("_", " ").capitalize() }}</label>
-                {% if filter_type == 'mapping'  %}
-                    <div>
-                        {% for data in sorted(adv_filter_data[filter_type]) %}
-                            {% if data %}
-                               <label class="radio-inline">
-                                  <input type="radio" class="filterCheckbox" name="mapping" onchange="modifyMapping(this, '{{ filter_type }}', '{{ data }}');"> {{ data }}
-                               </label>
-                            {% end %}
-                        {% end %}
-                        <label class="radio-inline">
-                            <input type="radio" class="filterCheckbox" name="mapping" onchange="modifyMapping(this, '{{ filter_type }}', '{{ "" }}');" checked> OWTF
-                        </label>
-                    </div>
-                {% else %}
-                    <div>
-                        {% for data in sorted(adv_filter_data[filter_type]) %}
-                            {% if data %}
-                               <label class="checkbox-inline">
-                                  <input type="checkbox" class="filterCheckbox" onchange="modifyFilter(this, '{{ filter_type }}', '{{ data }}');"> {{ data }}
-                               </label>
-                            {% end %}
-                        {% end %}
-                    </div>
-                {% end %}
-                <hr>
-            {% end %}
-      </div>
-      <div class="modal-footer">
-        <button type="button" class="btn btn-danger" onclick="clearFilters();" data-dismiss="modal" aria-hidden="true">Clear Filters!</button>
-      </div>
-    </div>
-  </div>
-</div>
-<!-- End Plugin Output Filter Modal -->
-
 <script>
-// All api urls needed for calls, these are provided by the server
-var mySpace = {
-                    target_api_url:"{{ target_api_url }}",
-                    targets_ui_url:"{{ targets_ui_url }}",
-                    poutput_ui_url:"{{ poutput_ui_url }}",
-                    filter_data:{},
-                    plugins_api_url:"{{ plugins_api_url }}?group=web&group=network",
-                    worklist_api_url:"{{ worklist_api_url }}",
-                    transaction_log_url:"{{ transaction_log_url }}",
-                    url_log_url:"{{ url_log_url }}",
-                    target_config:{},
-                    sessions_ui_url: "{{ sessions_ui_url }}"
-                  };
-
-function clearFilters() {
-    mySpace.filter_data = {};
-    $(".filterCheckbox").attr("checked", false);
-}
-
-function modifyFilter(checkbox, filter_type, data){
-    if (typeof mySpace.filter_data[filter_type] === 'undefined') {
-        mySpace.filter_data[filter_type] = [];
+    var mySpace = {
+        target_id: "{{ target_id}}",
+        worklist_api_url: "{{ worklist_api_url }}",
+        plugins_api_url: "{{ plugins_api_url }}?group=web&group=network",
```

Comment:
@delta24 Actually this is not hard-coded by me. I just copied it from plugin_report.html. I think no need to have this hard code thing now. I will remove it.